crates/api_models/src/payouts/additional_info.rs

```diff
@@ -0,0 +1,186 @@
+use common_utils::new_type::{
+    MaskedBankAccount, MaskedBic, MaskedEmail, MaskedIban, MaskedPhoneNumber, MaskedRoutingNumber,
+    MaskedSortCode,
+};
+use masking::Secret;
+use serde::{Deserialize, Serialize};
+use utoipa::ToSchema;
+
+use crate::enums as api_enums;
+
+#[derive(Eq, PartialEq, Clone, Debug, Deserialize, Serialize, ToSchema)]
+pub enum AdditionalPayoutMethodData {
+    Card(Box<CardAdditionalData>),
+    Bank(Box<BankAdditionalData>),
+    Wallet(Box<WalletAdditionalData>),
+}
+
+#[derive(Eq, PartialEq, Clone, Debug, Serialize, Deserialize, ToSchema)]
+pub struct CardAdditionalData {
```

Comment:
Possible to re-use this struct from payments? Or is there anything different here for payins vs payouts?

Reply:
struct from payment have additional payment fields which are optional but I think it would be better to have our own struct to contain any specific data for card in payouts if required